### ISSUE TITLE
Feature request: suppressible logging output

### ISSUE THREAD

**Issue by ZacSweers** [dropbox/dependency-guard#82]
Currently there are some noisy logs that can't be suppressed and run even if there's no changes. If the task is successful, I'd expect it to just pass without extra logging. Could this be made configurable?

**Comment by handstandsam** [comment]
Sounds reasonable.  This can definitely be planned in the future.

**Comment by ZacSweers** [comment]
Want a PR?

**Comment by handstandsam** [comment]
Would love it.  Working with Pinterest right now to get 0.4.x to work in their codebase, and finally thing I have https://github.com/dropbox/dependency-guard/pull/90 under control to release `0.4.3`.  If that works for them, then I think we're good to start with `0.5.0`.

TL;DR - Wait until https://github.com/dropbox/dependency-guard/pull/90 merges, and can create a PR on `main` then 🙇

**Comment by handstandsam** [comment]
I was about to ask "what do you want suppressed", so having a PR would be helpful to remove back & forth.

**Comment by handstandsam** [comment]
You're good to go.  I'm going to end up bumping `main` to `0.5.0-SNAPSHOT` here in a bit.

Merged #90, I've validated the fixes in 3 large projects including a mono repo.  Feeling good about it, so `0.4.3` will HOPEFULLY be the last release of the "Adding Configuration Cache" `0.4.x` release series 😭 .
